### ISSUE TITLE
Clean up hack for Crio-1.17.3+ bug

### ISSUE THREAD

**Issue by medyagh** [kubernetes/minikube#8861]
While I was trying to rebuild our current Kicbase image without ANY change, I noticed All the crio tests fail
https://github.com/kubernetes/minikube/pull/8854

Failed Test | Duration
-- | --
TestStartStop/group/crio/serial/SecondStart | 762.94
TestStartStop/group/crio/serial/UserAppExistsAfterStop | 541.09
TestStartStop/group/crio/serial/AddonExistsAfterStop | 541.04
TestStartStop/group/crio/serial/Pause

attached is the full test log in text
[testout.txt](https://github.com/kubernetes/minikube/files/4985980/testout.txt)

created an issue in the cri-o repo https://github.com/cri-o/cri-o/issues/4027 unfortunately the 1.17.3  package has vanished from the repo magically without any notice !

CC: @afbjorklund 

Example of error of TestStartStop/group/crio/serial/SecondStart

```
Unfortunately, an error has occurred:
		timed out waiting for the condition
	
	This error is likely caused by:
		- The kubelet is not running
		- The kubelet is unhealthy due to a misconfiguration of the node in some way (required cgroups disabled)
	
	If you are on a systemd-powered system, you can try to troubleshoot the error with the following commands:
		- 'systemctl status kubelet'
		- 'journalctl -xeu kubelet'
	
	Additionally, a control plane component may have crashed or exited when started by the container runtime.
	To troubleshoot, list all containers using your preferred container runtimes CLI, e.g. docker.
	Here is one example how you may list all Kubernetes containers running in docker:
		- 'docker ps -a | grep kube | grep -v pause'
		Once you have found the failing container, you can inspect its logs with:
		- 'docker logs CONTAINERID'
	
	stderr:
		[WARNING FileContent--proc-sys-net-bridge-bridge-nf-call-iptables]: /proc/sys/net/bridge/bridge-nf-call-iptables does not exist
		[WARNING SystemVerification]: failed to parse kernel config: unable to load kernel module: "configs", output: "modprobe: FATAL: Module configs not found in directory /lib/modules/4.9.0-13-amd64\n", err: exit status 1
		[WARNING Service-Kubelet]: kubelet service is not enabled, please run 'systemctl enable kubelet.service'
	error execution phase wait-control-plane: couldn't initialize a Kubernetes cluster
	
	run
	k8s.io/minikube/pkg/minikube/bootstrapper/kubeadm.(*Bootstrapper).init
		/home/jenkins/actions-runner/_work/minikube/minikube/pkg/minikube/bootstrapper/kubeadm/kubeadm.go:240
```

**Comment by tstromberg** [comment]
Related: #8890

**Comment by sharifelgamal** [comment]
Upstream 1.17.4 behavior issue: https://github.com/cri-o/cri-o/issues/4035

**Comment by medyagh** [comment]
we ended up making a Hack to restart CRIO each time, this is a temproary hack till CRIO upstream is fixed https://github.com/cri-o/cri-o/issues/4027

**Comment by afbjorklund** [comment]
> we ended up making a Hack to restart CRIO each time, this is a temproary hack till CRIO upstream is fixed [cri-o/cri-o#4027](https://github.com/cri-o/cri-o/issues/4027)

The actual bug for cri-o is https://github.com/cri-o/cri-o/issues/4035, the other issue is just about the deletion of the 1.17.3 package

It can still be rebuilt from the OBS sources, but by the time it was made available you had already upgraded to 1.18.3 instead

https://build.opensuse.org/package/show/devel:kubic:libcontainers:stable:cri-o:1.17:1.17.3/cri-o

Seems like the CRI-O project have changed from one repository to one subproject per release, so you can pin versions now...

https://build.opensuse.org/project/subprojects/devel:kubic:libcontainers:stable:cri-o

**Comment by sharifelgamal** [comment]
I have a PR open (#9501) that reverts this change, but the crio tests are still failing in a way that concerns me, so I need to do some more investigation. Do we need to update the version of crio we're using?

**Comment by afbjorklund** [comment]
I think the bug is fixed in PR #9629 (by upgrading runc version) - if that is correct we can do the suggested revert
